header.html

```diff
@@ -13,26 +13,26 @@
 </head>
 <body>
     <header class="header">
-        <img src="img/header-shapesmobile@2x.svg" alt="" class="bg-shapes">
-            <nav>
-                <ul class="navigation">
-                    <li class="logo">
-                        <a href="#">My Logo</a>
-                    </li>
+            <nav class="navigation">
+                <p class="logo">
+                    <a href="#">My Logo</a>
+                </p>
+                <ul class="nav-items">
+                    <li class="nav-link"><a href="#">Portfolio</a></li>
+                    <li class="nav-link"><a href="#">About</a></li>
+                    <li class="nav-link"><a href="#">Contact</a></li>
                     <li class="menu-icon">
                         <a href=""><i class="fa-solid fa-bars"></i></a>
                     </li>
                 </ul>
             </nav>
-    </header>
+    </header>  
+    <main>
         <!--Front Page-->
         <section class="dev">
-            <h1>I’m Mariane
-                Glad to see you!
-            </h1>
-                <p>
-                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-                </p>
+            <div class="dev-child">
+                <h1>I’m Justice Glad to see you!</h1>
```

Comment:
- [ ] Kindly note that you are required to stick as much as possible to the provided Figma design in fonts, colors, etc. The font size used is 6rem which makes it around 96px. Kindly change it to 48px as specified in the Figma design